django/db/models/functions.py

```diff
@@ -47,7 +47,8 @@ def as_sqlite(self, compiler, connection):
         return super(ConcatPair, self).as_sql(compiler, connection)
 
     def as_mysql(self, compiler, connection):
-        self.coalesce()
+        self.function = 'CONCAT_WS'
```

Comment:
I would like a brief comment here mentioning the use of concat_ws due to the handling of nulls.